### PR TITLE
Enabling 32 bits downgrade of our environment.

### PR DESCRIPTION
**Pull request by nicolasnoble**

### CODE REVIEW

**Comment by nicolasnoble** [comment]
I actually have a better idea for this one.

**Comment by nicolasnoble** [comment]
Made something more generic.

**Comment by nicolasnoble** [comment]
As a more descriptive explanation, this allows us to specify a platform with a name such as "linux;i386;gcc-4.4" for example. The script will extract the various options and we can start taking decisions based on that inside the script.

File: tools/jenkins/run_jenkins.sh [hunk 1]
```diff
@@ -37,6 +37,26 @@
 # NOTE: No empty lines should appear in this file before igncr is set!
 set -ex -o igncr || set -ex
 
+arch=`uname -m`
```
Comment (jtattermusch) [inline tools/jenkins/run_jenkins.sh]: Please add a comment what this does.

**Comment by nicolasnoble** [comment]
New version up and green.

**Pull request merged**